homepairs/homepairsUI/src/elements/Forms/GoogleInputForm.tsx

```diff
@@ -1,10 +1,12 @@
 import { GoogleAutoComplete } from 'react-native-google-autocomplete';
 import React from 'react';
-import { Text, View, TextInput, ViewStyle, StyleSheet, ScrollView } from 'react-native';
+import { AppState } from 'homepairs-types';
+import { connect } from 'react-redux';
+import { Text, View, TextInput, ViewStyle, StyleSheet, ScrollView, AsyncStorage } from 'react-native';
 import {HelperText} from 'react-native-paper';
 import {FontTheme} from 'homepairs-base-styles';
 import * as BaseStyles from 'homepairs-base-styles';
-import LocationItem from '../LocationItem/LocationItem';
+import {LocationItem} from 'homepairs-elements';
```

Comment:
This risks us for dependency cycles. A modules referenced in an alias should never reference the alias.

Reply:
Fixed